### ISSUE TITLE
Document the "Features" of a Canopy Site.

### ISSUE THREAD

**Issue by markpbaggett** [canopy-iiif/docs#31]
[] Document the the Works Page (/works/{slug})
[] Document the Search (/works)
[] Document Home (/)
[] Document Metadata (/metadata)

**Comment by mathewjordan** [comment]
Yep, see #18. I also think this could be where the SSR API routes are documented.
